src/assets/bioData.js

```diff
@@ -12,15 +10,15 @@ import Rana from './Rana.png';
 import Tazin from './Tazin.png';
 import James from './james.jpeg'
 import Sebra from './sabra.jpeg'
-export const data =[
+export const bioData =[
     {
```

Comment:
I changed the name of this to bioData so that I could set the state variable to `data` in `App.jsx` without confusion